client/landing/subscriptions/components/tabs-switcher/tabs-switcher.tsx

```diff
@@ -3,6 +3,7 @@ import { SubscriptionManager } from '@automattic/data-stores';
 import { useLocale } from '@automattic/i18n-utils';
 import { useTranslate } from 'i18n-calypso';
 import { Navigate, Route, Routes, useLocation, useNavigate } from 'react-router-dom';
+import { useIsLoggedIn } from 'calypso/../packages/data-stores/src/reader/hooks';
```

Comment:
You should not import the hook that way. The hook should be exported [here](https://github.com/Automattic/wp-calypso/blob/1701398d37cca5972323860dd4dd504fdd4b3873/packages/data-stores/src/reader/index.ts#L1), and then you can use it via the already imported here `SubscriptionManager`.

Reply:
Oh, nice. Doing the change 👍

Reply:
Very true! Change done 👍